tests/unit/amo/components/TestAddonCompatibilityError.js

```diff
@@ -40,50 +44,86 @@ describe(__filename, () => {
     });
   };
 
-  function render({ ...props }) {
+  const getClientCompatibilityNonFirefox = sinon.stub().returns(
```

Comment:
Why do you need to use `sinon.stub()`? Can't you simply create a function?

Reply:
I suppose I could. I seem to recall a PR from some time ago where I was asked to use `sinon` rather than create my own mock function. I think it was probably @kumar303 who asked me to do that. You don't see any value in doing that consistently?

Reply:
You call the original function below so it does not really mock anything, and I don't see any assertion on this function. So keep it simple.